_posts/2016-08-09-the-tech-behind-live-auction-integration.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "The Tech Behind Live Auction Integration"
+date: 2016-08-09 11:30
+comments: true
+author: Alan Johnson
```

Comment:
> You want to add yourself to the authors in the config, will add the following to the README

## Adding an Author

Authors are key-value stored, so you'll need to give yourself a key inside [_config.yml](_config.yml) - for example:

```yaml
  joey:
    name: Joey Aghion
    github: joeyAghion
    twitter: joeyAghion
    site: http://joey.aghion.com
```

> Everything but name is optional.